process/smartContract/process.go

```diff
@@ -1952,8 +1952,14 @@ func (sc *scProcessor) penalizeUserIfNeeded(
 		sc.gasHandler.SetGasPenalized(vmOutput.GasRemaining, txHash)
 	}
 
-	vmOutput.ReturnMessage += fmt.Sprintf("%s: gas needed = %d, gas remained = %d",
-		TooMuchGasProvidedMessage, gasUsed, vmOutput.GasRemaining)
+	if sc.flagCleanUpInformativeSCRs.IsSet() {
+		vmOutput.ReturnMessage += fmt.Sprintf("%s: gas needed = %d, gas remained = %d",
+			TooMuchGasProvidedMessage, gasUsed, vmOutput.GasRemaining)
+	} else {
+		vmOutput.ReturnMessage += fmt.Sprintf("%s for processing: gas needed = %d, gas remained = %d",
```

Comment:
I would add a test here just for backwards compatibility reasons. As to not be tempted to be changed by someone else.

Reply:
no-one is tempted to change a thing in the scProcessor ==))

Reply:
This is a good one! :1st_place_medal: